File: Java LN/Logica_JAVA/Ejercicios_clase_21_de_Abril/Ejercicio2.java
public class Ejercicio2 {
    public static void main(String[] args){
        int suma = 0;
        int resta = 0;
        int multiplicacion = 0;
        float division = 0;
        int[] numeros_1 = {1,2,3,4,5};
        int[] numeros_2 = {1,2,3,4,5};
        for(int i = 0; i < numeros_1.length; i++){
            for(int j = 0; j < numeros_2.length; j++){
                suma = numeros_1[i] + numeros_2[j];
                resta = numeros_1[i] - numeros_2[j];
                multiplicacion = numeros_1[i] * numeros_2[j];
                division = (float)numeros_1[i] / (float)numeros_2[j];
                System.out.println("\nLa suma de " + numeros_1[i] + " y " + numeros_2[j] + " es igual a: " + suma);
                System.out.println("La resta de " + numeros_1[i] + " y " + numeros_2[j] + " es igual a: " + resta);
                System.out.println("La multiplicacion de " + numeros_1[i] + " y " + numeros_2[j] + " es igual a: " + multiplicacion);
                System.out.println("La division de " + numeros_1[i] + " y " + numeros_2[j] + " es igual a: " + division + "\n");
            }
        }
    }
}

File: CSharp/Ejercicios Oscar 21 de Abril/Ejercicio2.cs
using System;

namespace Ejercicio2{
    class Program{
        static void Main(String[] args){
            int suma, resta, multiplicacion;
            float division;
            int[] numeros1 = {4,5,6,7,8};
            int[] numeros2 = {9,8,7,5,6};
            for(int i = 0; i < numeros1.Length; i++){
                Console.WriteLine(suma1);
                for(int j = 0; j < numeros2.Length; j++){
                    suma = numeros1[i] + numeros2[j];
                    resta = numeros1[i] - numeros2[j];
                    multiplicacion = numeros1[i] * numeros2[j];
                    division = (float)numeros1[i] / (float)numeros2[j];
                    Console.WriteLine("\nLa suma de {0} y {1} es: {2}", numeros1[i], numeros2[j], suma);
                    Console.WriteLine("La resta de {0} y {1} es: {2}", numeros1[i], numeros2[j], resta);
                    Console.WriteLine("La multiplicacion de {0} y {1} es: {2}", numeros1[i], numeros2[j], multiplicacion);
                    Console.WriteLine("La division de {0} y {1} es: {2} \n", numeros1[i], numeros2[j], division);
                }
            }
        }
    }
}